src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.netcoreapp.cs

```diff
@@ -0,0 +1,255 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.Linq;
+using System.Net.Test.Common;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Xunit;
+
+namespace System.Net.Http.Functional.Tests
+{
+    public abstract partial class HttpClientHandlerTest : HttpClientHandlerTestBase
```

Comment:
Why not just make this a separate class? Something like HttpClientHandlerTest_TrailingHeaders

Reply:
I thought when we add new NetCore only APIs later, we can put the new tests in this file as well.

Not sure what's the best practice here.

Reply:
@stephentoub Can you comment here?  In the past, we've used ".netcoreapp.cs" to separate out NetCoreApp only APIs (product source and tests).

Reply:
And we already have examples of that here with "HttpClientTest.netcoreapp.cs"

Reply:
I think there are multiple comments being made here.

In the past, we had gigantic test classes with lots of tests covering lots of different things.  We've started moving away from that, refactoring existing tests into smaller, focused classes.  And I think that's what Geoff is suggesting here: rather than adding to HttpClientHandlerTest, create a new HttpClientHandlerTest_TrailingHeaders type that contains tests focused just on trailing headers.  That sounds fine to me.

Separately, we've generally used a .netcoreapp.cs suffix when adding test files specific to netcoreapp.  That's fine, too, though it's more relevant when a class is split across multiple files, one of which is compiled in always and one of which is compiled in only when building for netcoreapp.  I'm fine with adding the suffix in this case as well, but it's not strictly necessary.

Reply:
> We've started moving away from that, refactoring existing tests into smaller, focused classes.

Ok, will separate the tests.

> I'm fine with adding the suffix in this case as well, but it's not strictly necessary.

Will not add suffix as Framework does not support trailers anyway.